ntc_templates/templates/fsas_sir_show_ether.textfsm

```diff
@@ -0,0 +1,30 @@
+Value Required ETHER_GROUP (\d+)
+Value Required ETHER_PORT (\d+)
+Value DESCRIPTION (\S+)
```

Comment:
Is it possible for a description on this platform to accept a multi-word description?
If yes, we may want to update the regex.

```suggestion
Value DESCRIPTION (.+?)
```